docs/demo-bootstrapper-output.md

```
@@ -550,3 +555,404 @@ rerun options are worded to distinguish a genuinely different tradeoff instead:
+[INFO] REQ-005.5: dependency source diff computed -- ~pipreqs.diff.txt
+[INFO] REQ-005.12: autopep723 discovery merge complete.
+```
+
+**A real quirk worth flagging so it isn't misread**: the bootstrap log around this point also shows
+`DEP_FINAL_COUNT=0` even though `colorama` is a genuine, real dependency that gets installed a few
+steps later -- this count is taken BEFORE `requirements.auto.txt` is copied into `requirements.txt`
+a few lines further down (`run_setup.bat:1330-1358`), so `DEP_FINAL_COUNT=0` here does not mean
+"pipreqs found nothing"; it's a pre-existing ordering quirk in the log sequence, not a bug in this
+run. `~setup.log`-only (pipreqs's raw output is redirected to a SEPARATE file entirely,
+`~pipreqs_direct.log`, not even `~setup.log` -- a real user never sees any of the following):
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)